docs/services/sdk/overview.md

```diff
@@ -18,8 +18,50 @@ Add the `@restatedev/restate-sdk` dependency to your NodeJS project to start dev
 The Typescript SDK uses the `protobufjs` and `ts-proto` dependencies to work with the Protobuf-generated Typescript code.
 So add these as well to your project.
```

Comment:
I guess this is something one only needs for the gRPC based API.

Reply:
I will move it down into the handler API section